### PR TITLE
co/dcos-61136 (2.0.1 release notes) 

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.mesosphere.com/browse/DCOS-61136

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# Fixed and Improved Issues in DC/OS 2.0.1
+<!-- The issues that have been fixed and improved in DC/OS 2.0.1 are grouped by feature, functional area, or component.  -->
+- Fixed an issue where unfinished deployments prevent DCOS instances running on Marathon 1.9.100 from migrating properly.
```
Comment (timcharper) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md]:
```suggestion
- Fixed an issue where unfinished Marathon deployments would prevent a successful upgrade to DC/OS 2.0.```

**Comment by cmastr** [comment]
> DC/OS 2.0.1 was released on 20 November 2019. --> DC/OS 2.0.1 was released on 22 November 2019.
> 
> Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). --> Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://d2iq.mesosphere.com/s/downloads).

ooh that link needs to be changed a lot of places! good catch.

File: pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.1/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# Fixed and Improved Issues in DC/OS 2.0.1
+<!-- The issues that have been fixed and improved in DC/OS 2.0.1 are grouped by feature, functional area, or component.  -->
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md]: We would also need to add the release note for mesos memory leak issue. cc @greggomann

Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md]: Can one of you provide the JIRA ticket number?

Comment (greggomann) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md]: https://jira.mesosphere.com/browse/DCOS-61228

Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.1/index.md]: ty. resolved.

**Pull request merged**